www/editcomp

```diff
@@ -1461,6 +1462,50 @@ function activateListeners(divno, model)
         sortDivByPlace('divModel' + divno, window["divVals"+divno]);
     });
 
+    document.querySelector('#addByTag' + divno).addEventListener('click', async function(event) {
+        const tag = prompt("Tag name?");
+
+        const dialog = document.createElement("dialog");
+        dialog.innerText = "Loading";
+        document.body.append(dialog);
+        dialog.showModal();
+
+        const url = '/search?' + new URLSearchParams({
+            searchbar: 'tag:' + tag,
+            xml: 1,
+            pg: 'all',
+            sortby: 'ttl',
+        }).toString();
```

Comment:
I think `toString()` isn't needed.

And this might be more readable:
```javascript
const params = new URLSearchParams(...);
...
await fetch(`/search?{$params}`);
```